src/components/ActionLink/ActionLink.vue

```diff
@@ -75,11 +75,11 @@ export default {
 			default: '#',
 			required: true,
 			validator: value => {
-				// href is either an anchor or a valid url
+				// href is either an anchor or a valid url starting with a scheme or an absolute path
 				try {
 					return new URL(value)
 				} catch (error) {
-					return value.startsWith('#')
+					return value.startsWith('#') || value.startsWith('/')
```

Comment:
relative? :P

Reply:
how could we check that? :thinking:

Reply:
no. a URL that start with `/` is *relative*. One that starts with a scheme or the host is *absolute*

Reply:
The PR description (and the source code comment) should say *Also allow relative url*

Reply:
:see_no_evil: